src/main.js

```diff
@@ -236,10 +267,11 @@ function createStudyTitle(div) {
  * Modules with grade F are not counted in the average.
  * A second average is displayed, where the modules with grade F are taken into account.
  */
-function createAverageMarkTitle(div) {
-    let average = Number(_Student.totalNumericMark / _Student.numberOfNumericMarks).toFixed(2);
-    let averageWithF = Number(_Student.totalNumericMarkWithF / _Student.numberOfNumericMarksWithF).toFixed(2);
-    Helpers.addTitleToDocument(div, 'Noten Ø: ' + average + ' (Ø mit F: ' + averageWithF + ')')
+async function createAverageMarkTitle(div) {
```

Comment:
I you wouldn't load the JSON for every call of `i18n.getMessage()`, this method wouldn't need to be async.

Reply:
done